src/WebJobs.Script/Workers/SharedMemoryDataTransfer/IMemoryMappedFileAccessor.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO.MemoryMappedFiles;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.SharedMemoryDataTransfer
+{
+    public interface IMemoryMappedFileAccessor
```

Comment:
If all these new interfaces/types are only used internally in this assembly (which it seems like they are), can you declare them all internal? That way we don't have to worry about breaks if you need to change the interface definitions in future revisions.

Reply:
You mean putting `internal` for these methods in the interface?

I'm getting the following error:
CS8703: The modifier `internal` is not valid for this item in C# 7.2. Please use language version 8.0 or greater.